protocol/contracts/admin/VestingPayments.sol

```diff
@@ -224,19 +228,19 @@ contract VestingPayments is IVestingPayments, PaymentBase {
     function _calculateReleasableAmount(
           VestingSchedule storage _schedule
     ) private view returns (uint256) {
-        // if account schedule is revoked, return the persisted checkpoint vested amount. This also avoids an arithmetic underflow
-        return _schedule.revoked ? revokedAccountsReleasable[_schedule.recipient] : _calculateTotalVestedAt(_schedule, uint40(block.timestamp)) - _schedule.distributed;
+        // if account schedule is revoked, return the persisted checkpoint claimable vested delta. This also avoids an arithmetic underflow
+        return _schedule.revoked ? _schedule.revokedReleasable : _calculateTotalVestedAt(_schedule, uint40(block.timestamp)) - _schedule.distributed;
     }
 
     function _calculateTotalVestedAt(
         VestingSchedule memory _schedule,
         uint40 _releaseTime
-    ) private view returns (uint256) {
+    ) private pure returns (uint256) {
         if (_schedule.amount == 0) { return 0; }
         // below cliff
         if (_releaseTime <= _schedule.cliff) { return 0; }
-        // if revoked, return releasable amount at time of revoke
-        if (_schedule.revoked) { return revokedAccountsReleasable[_schedule.recipient]; }
+        // if revoked, return total vested at time of revoke (distributed + checkpointed delta)
+        if (_schedule.revoked) { return _schedule.revokedReleasable + _schedule.distributed; }
```

Comment:
Given this tweak to be `revokedReleasable + distributed`,  I think that means the line 232 doesn't need the ternary check now? It can just simplify to:

```
return _calculateTotalVestedAt(_schedule, uint40(block.timestamp)) - _schedule.distributed;
```
?

Reply:
moved to https://github.com/TempleDAO/temple/pull/1169/files#r2312699213